articles/sentinel/notebooks.md

```diff
@@ -95,46 +171,76 @@ Once your deployment is complete, you can launch notebooks in your new Azure ML
+
+1. Select the notebook you want to use, and then select **Save Notebook** to clone the notebook template into your own notebook project. You will then have an option to rename the notebook or keep the default name. 
+
+    > [!div class="mx-imgBorder"]
+    > ![save notebook](./media/notebooks/sentinel-notebooks-save.png)
 
-1. Select the notebook you want to use, and then select **Launch Notebook** to clone and configure the notebook into a new Azure Notebooks project that connects to your Azure Sentinel workspace. When the process is complete, the notebook opens within Azure Notebooks for you to run.
+1. Once the notebook is saved to your default Azure ML workspace, on the same view, select **Launch notebook**. 
 
     > [!div class="mx-imgBorder"]
-    > ![select notebook](./media/notebooks/sentinel-azure-notebooks-select.png)
+    > ![launch notebook](./media/notebooks/sentinel-notebooks-launch.png)
 
-1. Under AzureML Workspace, select your Azure ML workspace, and then select **Launch**.
+1. A new page, your Azure ML studio, is open.
 
     > [!div class="mx-imgBorder"]
-    > ![launch notebook](./media/notebooks/sentinel-azure-notebooks-launch.png)
+    > ![aml notebook](./media/notebooks/sentinel-notebooks-on-aml.png)
+
+1. Select a compute instance. The compute is used for your notebook server. 
 
-1. Select a compute instance. If you don't have a compute instance, do the following:
-    1. Select the plus sign (+) to start the **Create compute instance** wizard.
+    * If you don't have a compute instance, do the following:
+        
+        a. In the notebook or script toolbar, to the right of the Compute dropdown, select **+ New Compute**. Depending on your screen size, this may be located under a ... menu.
 
         > [!div class="mx-imgBorder"]
-        > ![start compute instance wizard](./media/notebooks/sentinel-azure-notebooks-compute-wizard.png)
+        > ![start compute instance wizard](./media/notebooks/sentinel-notebooks-new-compute.png)
 
-    1. On the **Select virtual machine** page, provide the required information, and then select **Next**.
+        b. On the **Configured required settings** page, name a compute, choose a Virtual Machine size based on your workload types, and then select **Next**.
+
+        Start with one with the **General Purpose** category first if you need compute to only test the notebooks.
+        Only you can see and use the compute instances you create. Your User files are stored separately from the VM and are shared among all compute instances in the workspace.
 
         > [!div class="mx-imgBorder"]
-        > ![Select compute instance VM](./media/notebooks/sentinel-azure-notebooks-compute-create.png)
+        > ![Select compute instance VM](./media/notebooks/sentinel-notebooks-compute-select.png)
+    
+    * If you have a stopped compute instance, select **Start compute** to the right of the Compute dropdown. Depending on your screen size, this may be located under a ... menu.
```

Comment:
Please either replace the ellipsis (...) with appropriate text, or remove the whole "Depending on your screen size..." sentence.